queries/plugins/text_to_num.py

```diff
@@ -0,0 +1,573 @@
+"""
+
+    Greynir: Natural language processing for Icelandic
+
+    Natural language number parsing.
+
+    Copyright (C) 2021 Miðeind ehf.
+
+       This program is free software: you can redistribute it and/or modify
+       it under the terms of the GNU General Public License as published by
+       the Free Software Foundation, either version 3 of the License, or
+       (at your option) any later version.
+       This program is distributed in the hope that it will be useful,
+       but WITHOUT ANY WARRANTY; without even the implied warranty of
+       MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+       GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/.
+
+
+    Nonterminal for a single number written in natural language.
+    Returns its value in the list result["numbers"].
+
+"""
+
+# TODO: Add to Vocab.conf numbers not in BÍN (see bottom of file)
+# TODO: Deal with cases and genders of numbers (/fall/kyn/tala)
+# TODO: Allow "tólf hundruð þúsund"? Allow "milljón milljónir"?
```

Comment:
Yes, both "tólf hundruð þúsund" and "milljón milljónir" should preferably be allowed.